scripts/deployment/deploy-documentation.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+# Exit the script on any command with non 0 return code
+set -e
+
+# Go to project root
+cd "$(dirname "$0")"
+cd ../../
+
+if [ "${DRONE}" = "true" ]; then
+  git config --global user.email "DIGIT-INNO@ec.europa.eu";
+  git config --global user.name "HAL Patch INNO";
+fi
+
+# Deploy
+./node_modules/.bin/gh-pages \
```

Comment:
Why not using `npx` here?

Reply:
Because this part of the script is used only by the CI and I didn't want to change anything and spend hours debugging something I can't reproduce locally.